tests/playwright/pages/Dashboard/common/LeftSidebar/index.ts

```
@@ -120,4 +160,56 @@ export class LeftSidebarPage extends BasePage {
 
     await this.rootPage.waitForTimeout(3500);
   }
+
+  async miniSidebarActionClick({
+    type,
+    fallback,
+  }: {
+    type: 'ws' | 'base' | 'cmd-k' | 'teamAndSettings' | 'integration' | 'feeds' | 'notification' | 'userInfo';
+    fallback?: () => Promise<void>;
+  }): Promise<boolean | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)